dm/loader/loader.go

```diff
@@ -441,8 +441,8 @@ type Loader struct {
 	// to calculate remainingTimeGauge metric, map will be init in `l.prepare.prepareDataFiles`
 	dbTableDataTotalSize        map[string]map[string]*atomic.Int64
 	dbTableDataFinishedSize     map[string]map[string]*atomic.Int64
-	dbTableDataLastFinishedSize map[string]map[string]int64
-	dbTableDataLastUpdatedTime  time.Time
+	dbTableDataLastFinishedSize map[string]map[string]*atomic.Int64
```

Comment:
would better to implement in object oriented way
```go
map[string]map[string]*tableData
type tableData struct {
  total atomic.Int64
  finished atomic.Int64
  lastFinished atomic.Int64
}
```

Reply:
we're going to deprecate Loader soon, currently I only want to fix bugs with least effort 😂